explorer/app/components/Export/components/ExportToTerra/exportToTerra.stories.tsx

```diff
@@ -3,11 +3,17 @@ import React from "react";
 import { ExportToTerra } from "./exportToTerra";
 
 export default {
+  argTypes: {
+    path: { control: "text" },
```

Comment:
Update from `path` to `params`. Then disable the arg `params: { control: { disable: true } },` because I'm not sure we will use the control given it is a URLSearchParm.